main.go

```diff
@@ -241,8 +241,10 @@ func main() {
 			xcodeBuildCmd.SetCustomOptions(options)
 		}
 
-		// Disabe indexing while building
-		xcodeBuildCmd.SetDisableIndexWhileBuilding(cfg.DisableIndexWhileBuilding)
+		// Disable indexing while building
+		if cfg.DisableIndexWhileBuilding {
+			xcodeBuildCmd.SetCustomBuildAction("COMPILER_INDEX_STORE_ENABLE=NO")
```

Comment:
this may override line 232.. what about adding a []string that collects all the custom action than apply it later on the command?

Reply:
Good catch! Should be fixed in https://github.com/bitrise-steplib/steps-xcode-build-for-simulator/pull/32/commits/fce02bbd95fb8cfcfffa0d9c28fc9e9b5bde498f